ddl/ddl_worker.go

```diff
@@ -69,6 +70,7 @@ type worker struct {
 	quitCh   chan struct{}
 	wg       sync.WaitGroup
 
+	statsHandle     unsafe.Pointer
```

Comment:
Use `domain` to get `statsHandle` will have a circle import problem. So stay the `statsHandle` in DDL worker.